src/main/resources/org/jenkinsci/plugins/github/config/GitHubPluginConfig/config.groovy

```diff
@@ -5,15 +5,12 @@ import com.cloudbees.jenkins.GitHubPushTrigger
 def f = namespace(lib.FormTagLib);
 
 f.section(title: descriptor.displayName) {
-    f.entry(title: _("Servers configs with credentials to manage GitHub integrations"),
-            description: _("List of GitHub Servers to manage hooks, set commit statuses etc."),
+    f.entry(title: _("GitHub Servers"),
             help: descriptor.getHelpFile()) {
         
         f.repeatableHeteroProperty(
                 field: "configs",
-                hasHeader: "true",
-                addCaption: _("Add GitHub Server Config"),
```

Comment:
I also don't like default "add" "delete" as they are not very useful on what they really do because of pure jelly form selection (it visible, but with all not aligned checkboxes not very convenient).